streamwebs_frontend/streamwebs/templates/streamwebs/index.html

```diff
@@ -1,21 +1,23 @@
+{% extends "streamwebs/base.html" %}
+{% load i18n %}
+
 <!DOCTYPE html>
 <html>
     <head>
         <title>StreamWebs</title>
     </head>
 
     <body>
```

Comment:
Also, we should add a {% block title %} to the base and child templates, so each child can set their own title, even though the <head> tag is only in the base template.

In the base:
```
<head>
{% block title %}{% endblock %}
```

and in the child:

```
{% block title %}
<title>The Page Title</title>
{% endblock %}
```